lib/util/chat/pages/chat_page.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/material.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/util/chat/message_bar.dart';
+import 'package:motis_mitfahr_app/util/chat/models/message.dart';
+import 'package:motis_mitfahr_app/util/chat/chat_bubble.dart';
+import 'package:motis_mitfahr_app/util/locale_manager.dart';
+import 'package:motis_mitfahr_app/util/profiles/profile_widget.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class ChatPage extends StatefulWidget {
+  const ChatPage({required this.rideId, required this.profile, super.key});
+  final int rideId;
+  final Profile profile;
+
+  @override
+  State<ChatPage> createState() => _ChatPageState();
+}
+
+class _ChatPageState extends State<ChatPage> {
+  late Stream<List<Message>> _messagesStream;
+
+  @override
+  void initState() {
+    _messagesStream = SupabaseManager.supabaseClient
+        .from('messages')
+        .stream(primaryKey: ['id'])
+        .eq('ride_id', widget.rideId)
+        .order('created_at')
+        .map((messages) => Message.fromJsonList(messages));
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _messagesStream.drain();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: ProfileWidget(
+          widget.profile,
+          isTappable: true,
+        ),
+      ),
+      body: StreamBuilder<List<Message>>(
+        stream: _messagesStream,
+        builder: (context, snapshot) {
+          if (snapshot.hasData) {
+            List<Message> messages = snapshot.data!;
+            return Column(
+              children: [
+                Expanded(
+                  child: messages.isEmpty
+                      ? Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: [
+                            Image.asset(
+                              'assets/chat_shrug.png',
+                              scale: 8,
+                            ),
+                            const SizedBox(height: 16),
+                            Text(
```

Comment:
this text and message shuld not be shown (or different) if there are no rides. or alternatively, do not even show the chat icon button at the top right in drive detail if there are no rides

Reply:
I think the second option is good i'll add that

Reply:
and where exactly does this message get shown? if there are no rides i shouldn't get to the chatPage

Reply:
What do you mean? Isn't that exactly what I said?

Reply:
no i meant where can i get there with the current implementation. sorry for the confusion

Reply:
well currently the drive chat icon is always shown?

Reply:
yes but that get's to the drive_chat_page not the chat_page

Reply:
but maybe we should change names xD

Reply:
Oh sorry yes I was confused... Forget what I said